### DISCUSSION TITLE
Education Pack problems (Can't review the discount) (Email Already used ???)

### DISCUSSION THREAD

**Discussion by ghost** [community/community#57323] [GitHub Education]
### Hi everyone,

I'm having trouble renewing my GitHub Education Pack subscription.

Initially he refused me telling me that "I had already used the pack", when I'm still enrolled in a university, so I should be able to renew the pack.

Now when I try to renew the pack, the page doesn't load and doesn't take any action.

Can you help me?

**Comment by samus-aran** [comment]
Hi @rgoth-code 

Are you still unable to re-apply with a new application? We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.
As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.

**Comment by ghost** [comment]
salve, come può vedere il mio problema riguardava il fatto che durante la
verifica dell'applicazione mi dà errore l'email associata all'istituto di
Tor Vergata. Mi dice che non posso richiedere l'applicazione con la stessa
email, pur chiedendomi di riverificare un account dove è già stato
applicato GitHub education.

**Comment by ghost** [comment]
Hi, as you can see, my problem was that during the verification of the
application, the email associated with the Tor Vergata institute gives me
an error. It tells me that I can't request the application with the same
email, even though it asks me to re-verify an account where GitHub
education has already been applied. I apologize but the previous email was
written in Italian.

**Reply by samus-aran** [reply]
Sorry, can I ask for some clarity. Is the issue you're experiencing is that you don't have a matching academic email to your selected school or that you had an academic email but it's associated to another GitHub account?

**Reply by ghost** [reply]
The problem encountered is that I used an academic email for the discount (Associated together with the main one to my only GitHub account), but when requesting the verification of the discount it tells me that I cannot request the discount with the institutional email "because already used". Yet it is the only one I have that can prove that I am a student of Tor Vergata.

**Reply by samus-aran** [reply]
Ok, it appears the email you want to use for your selected school is associated to another GitHub account. You can not reuse an academic email from multiple accounts. 

It’s not possible to transfer Global Campus benefits over from one GitHub account to another.
However, you are able to merge your accounts, but the account with the benefits would be the main account. You can add your non-academic email from the previous account to the account containing the academic email. If you desire you can then rename this account once merged. For more details check out out the [merging of your personal accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/merging-multiple-personal-accounts) documentation.

**Reply by ghost** [reply]
I've never used that email for another account, as that email is associated directly with my "only" github account. In fact, I've already registered GitHub Education for the first time and I've already done a review, but at the 2nd time it tells me that "I can't reuse the same email" when it's not possible to do anything else as my university only provides me with an email.

**Reply by ghost** [reply]
In fact (just made a try) if I try to access github with the institutional email it gives me access to the main account I have (the one I'm writing with).

**Reply by ghost** [reply]
Also I was asked on the verification page to "associate the institutional email with my main account" and that's what I did, yet I get this error:
![image_2023-07-26_234843934](https://github.com/community/community/assets/126329746/ee0f8e34-f12b-47eb-8e71-ae9d143fd629)
